txscript/data/script_tests.json

```diff
@@ -42,21 +42,516 @@
+["0 IF 0x4c 0x00 ENDIF 1", "", "NONE", "OK", "non-minimal PUSHDATA1 ignored"],
+["0 IF 0x4d 0x0000 ENDIF 1", "", "NONE", "OK", "non-minimal PUSHDATA2 ignored"],
+["0 IF 0x4c 0x00000000 ENDIF 1", "", "NONE", "OK", "non-minimal PUSHDATA4 ignored"],
+["0 IF 0x01 0x81 ENDIF 1", "", "NONE", "OK", "1NEGATE equiv"],
+["0 IF 0x01 0x01 ENDIF 1", "", "NONE", "OK", "OP_1  equiv"],
+["0 IF 0x01 0x02 ENDIF 1", "", "NONE", "OK", "OP_2  equiv"],
+["0 IF 0x01 0x03 ENDIF 1", "", "NONE", "OK", "OP_3  equiv"],
+["0 IF 0x01 0x04 ENDIF 1", "", "NONE", "OK", "OP_4  equiv"],
+["0 IF 0x01 0x05 ENDIF 1", "", "NONE", "OK", "OP_5  equiv"],
+["0 IF 0x01 0x06 ENDIF 1", "", "NONE", "OK", "OP_6  equiv"],
+["0 IF 0x01 0x07 ENDIF 1", "", "NONE", "OK", "OP_7  equiv"],
+["0 IF 0x01 0x08 ENDIF 1", "", "NONE", "OK", "OP_8  equiv"],
+["0 IF 0x01 0x09 ENDIF 1", "", "NONE", "OK", "OP_9  equiv"],
+["0 IF 0x01 0x0a ENDIF 1", "", "NONE", "OK", "OP_10 equiv"],
+["0 IF 0x01 0x0b ENDIF 1", "", "NONE", "OK", "OP_11 equiv"],
+["0 IF 0x01 0x0c ENDIF 1", "", "NONE", "OK", "OP_12 equiv"],
+["0 IF 0x01 0x0d ENDIF 1", "", "NONE", "OK", "OP_13 equiv"],
+["0 IF 0x01 0x0e ENDIF 1", "", "NONE", "OK", "OP_14 equiv"],
+["0 IF 0x01 0x0f ENDIF 1", "", "NONE", "OK", "OP_15 equiv"],
+["0 IF 0x01 0x10 ENDIF 1", "", "NONE", "OK", "OP_16 equiv"],
+
+["NOTIF test coverage"],
+["1 0", "NOTIF IF 1 ELSE 0 ENDIF ENDIF", "NONE", "OK"],
+["1 1", "NOTIF IF 1 ELSE 0 ENDIF ENDIF", "NONE", "OK"],
+["1 0", "NOTIF IF 1 ELSE 0 ENDIF ELSE IF 0 ELSE 1 ENDIF ENDIF", "NONE", "OK"],
+["0 1", "NOTIF IF 1 ELSE 0 ENDIF ELSE IF 0 ELSE 1 ENDIF ENDIF", "NONE", "OK"],
+["1", "NOTIF 0 NOTIF RETURN ELSE RETURN ELSE RETURN ENDIF ELSE 0 NOTIF 1 ELSE RETURN ELSE 1 ENDIF ELSE RETURN ENDIF ADD 2 EQUAL", "NONE", "OK"],
+["1", "NOTIF 0 ELSE 1 ELSE 0 ENDIF", "NONE", "OK", "Multiple ELSE's are valid and execution inverts on each ELSE encountered"],
+["0", "NOTIF 1 ELSE 0 ELSE ENDIF", "NONE", "OK"],
+["0", "NOTIF ELSE 0 ELSE 1 ENDIF", "NONE", "OK"],
+["0", "NOTIF 1 ENDIF", "NONE", "OK"],
+["0", "NOTIF 1 ELSE 0 ELSE 1 ENDIF ADD 2 EQUAL", "NONE", "OK"],
+["'' 0", "NOTIF SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ELSE ELSE SHA1 ENDIF 0x14 0x68ca4fec736264c13b859bac43d5173df6871682 EQUAL", "NONE", "OK"],
+["0", "NOTIF ELSE 1 ENDIF", "NONE", "ERR_EMPTY_STACK"],
+["0 0", "NOTIF IF 1 ELSE 0 ENDIF ENDIF", "NONE", "ERR_EVAL_FALSE"],
+["0 1", "NOTIF IF 1 ELSE 0 ENDIF ENDIF", "NONE", "ERR_EVAL_FALSE"],
+["1 1", "NOTIF IF 1 ELSE 0 ENDIF ELSE IF 0 ELSE 1 ENDIF ENDIF", "NONE", "ERR_EVAL_FALSE"],
+["0 0", "NOTIF IF 1 ELSE 0 ENDIF ELSE IF 0 ELSE 1 ENDIF ENDIF", "NONE", "ERR_EVAL_FALSE"],
+["0 NOTIF", "123", "NONE", "ERR_UNBALANCED_CONDITIONAL"],
+["NOP", "NOTIF 1 ENDIF", "NONE", "ERR_INVALID_STACK_OPERATION"],
```

Comment:
Since this test was added in this PR it needs a comment for what is being tested.